### DISCUSSION TITLE
After login redirection

### DISCUSSION THREAD

**Discussion by tvadovic** [typemill/typemill#490] [Q&A]
Hi,

can't figure out, how to redirect user after login to a page where the login was requested. there's just an option for redirection to frontend homepage. 

to be more precise, is is possible at all, when i restrict content to a logged in user on a specific page and provide a link in a restriction notice to a login page (/tm) to get him back to that page after succesfull login?

**Comment by trendschau** [comment]
This option does not exist yet, you can only determine a fixed page globally for a userrole. If you want to redirect to the referrer, then you have to store the referrer in the cookie for example and then redirect him on success. Might be a bit tricky to avoid redirect loops and similar problems. I could add an option like this into core, but to be honest I don't know when I find time for it. Maybe it is also possible with a plugin and use JavaScript, but not sure.

**Reply by tvadovic** [reply]
Fair enough, i will add a note to restricted pages for users to get back to where they were from homepage once logged in.
